tests/test_util.py

```diff
@@ -71,6 +72,26 @@ def test_write_pcap(self):
         pcap = PCAP()
         pcap.write_packets(proto_analyzer.messages, os.path.join(tempfile.gettempdir(), "test.pcap"), 1e6)
 
+    def test_write_pcapng(self):
```

Comment:
Thanks for adding a test! Feels like it should test something about the resulting PCAP file, though. Since we do not have an import maybe we can check file size or similar to ensure the file is actually written and has contents?

Reply:
Done: As we don't have PCAPNG importers as you said, we'll verify output just by checking file size, PCAPNG SHB type number and that all msg bytes were written somewhere inside output file